lib/yelp/fusion/singleton.rb

```diff
@@ -0,0 +1,43 @@
+module Yelp
+  module Fusion
+    module Singleton
+      # These methods were copied from the old yelp implementation
+      # https://github.com/Yelp/yelp-ruby
+
+      REQUEST_CLASSES = [Yelp::Fusion::Endpoint::Search,
+                         Yelp::Fusion::Endpoint::Business,
+                         Yelp::Fusion::Endpoint::Phone,
+                         Yelp::Fusion::Endpoint::Review,
+                         Yelp::Fusion::Endpoint::Transaction,
+                         Yelp::Fusion::Endpoint::Match].freeze
+
+      private
+
+      # This goes through each endpoint class and creates singletone methods
```

Comment:
singletone is should be spelled singleton

Reply:
+1